Pomodoro.Api/appsettings.json

```diff
@@ -13,6 +13,10 @@
     "SecurityKey": "PomodoroStrongSecurityKey",
     "Issuer": "PomodoroGroup_GL_BaseCamp",
     "Audience": "http://localhost:4200",
-    "ExpirationTimeMinutes":  60
+    "ExpirationTimeMinutes": 60
+  },
+  "GoogleSecrets": {
+    "ClientId": "513628381530-2nj8f4lftohck7drakp86h8ds427snk2.apps.googleusercontent.com",
```

Comment:
just a comment, better not to put such values here. As we're developing locally - it is understandable why you added them here, I think we can leave it like that for now. But in commercial projects when you'll be working with something like that - better to use secure storage for keys like Key Vault.
I am just pointing out as this situation occurs a lot even in commercial dev.

everything else looks awesome :) thanks